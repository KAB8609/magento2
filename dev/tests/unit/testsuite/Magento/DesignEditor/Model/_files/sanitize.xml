<?xml version="1.0" encoding="UTF-8"?>
<!--
/**
 * {license_notice}
 *
 * @category    Magento
 * @package     Magento_DesignEditor
 * @subpackage  unit_tests
 * @copyright   {copyright}
 * @license     {license_link}
 */
-->
<layout>
    <block name="root" class="Magento_Core_Block_Template" template="template.phtml">
        <block name="head" class="Magento_Page_Block_Html_Head">
            <action method="safeMethod">
                <argument name="arg" xsi:type="string">1</argument>
            </action>
        </block>
        <block name="custom.safe" class="Namespace_Module_Block_Safe"/>
        <container name="content">
            <block name="custom.not_safe" class="Namespace_Module_Block_NotSafe">
                <action method="setTemplate">
                    <argument name="arg" xsi:type="string">some_template.phtml</argument>
                </action>
            </block>
            <block class="Magento_Page_Block_Safe"/>
        </container>
    </block>
    <reference name="head">
        <action method="safeMethod">
            <argument name="arg" xsi:type="string">1</argument>
        </action>
    </reference>
    <reference name="custom.not_safe">
        <action method="rogueMethod">
            <argument name="block" xsi:type="string">1</argument>
        </action>
    </reference>
</layout>
