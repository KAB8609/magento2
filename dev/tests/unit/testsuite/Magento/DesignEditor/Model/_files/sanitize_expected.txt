<layout>
   <block name="root" type="Magento_Core_Block_Template" template="template.phtml">
      <block name="head" type="Magento_Page_Block_Html_Head">
         <action method="safeMethod">
            <arg>1</arg>
         </action>
      </block>
      <block name="custom.safe" type="Namespace_Module_Block_Safe"/>
      <container name="content">
         <block name="custom.not_safe" type="Magento_Core_Block_Template" template="Magento_DesignEditor::stub.phtml">%A</block>
         <block type="Magento_Page_Block_Safe"/>
      </container>
   </block>
   <reference name="head">
      <action method="safeMethod">
         <arg>1</arg>
      </action>
   </reference>
   <reference name="custom.not_safe">%A</reference>
</layout>
